src/pudl_archiver/archivers/ferc/xbrl.py

```diff
@@ -118,6 +138,55 @@ def __eq__(self, other: "FeedEntry"):
         return self.download_url == other.download_url
 
 
+class IndexedFilings(BaseModel):
```

Comment:
This class is just for convenience to make it a little bit easier to make sense of the objects getting passed around here. Substantively it's largely just moving some standalone code into the `classmethod` here.